indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -0,0 +1,163 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Form, Grid, Header, Modal} from 'semantic-ui-react';
+import {Form as FinalForm, Field} from 'react-final-form';
+import {ReduxFormField} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+
+class RoomEditModal extends React.Component {
+    static propTypes = {
+        onClose: PropTypes.func.isRequired
+    };
+
+    handleCloseModal = () => {
+        const {onClose} = this.props;
+        onClose();
+    };
+
+    renderModalContent = (fprops) => {
+        return (
+            <>
+                <Modal.Header>
+                    <Translate>Edit Room Details</Translate>
+                </Modal.Header>
+                <Modal.Content>
+                    <Form id="room-form" onSubmit={fprops.handleSubmit}>
+                        <Grid columns={3}>
+                            <Grid.Column>
+                                <Header>
+                                    <Translate>Contact</Translate>
+                                </Header>
+                                <Field name="owner"
+                                       component={ReduxFormField}
+                                       componentLabel={Translate.string('Owner')}
+                                       as={Form.Input} />
+                                <Field name="key"
+                                       component={ReduxFormField}
+                                       componentLabel={Translate.string('Where is the key?')}
+                                       as={Form.Input} />
```

Comment:
I think that gives you duplicate Form.Field wrapping. You can just use `as="input"` here (probably also add `type="text"`)